src/sync/installation.ts

```diff
@@ -45,7 +45,7 @@ export const getTargetTasks = (targetTasks?: TaskType[]): TaskType[] => {
 const getNextTask = async (subscription: Subscription, targetTasks?: TaskType[]): Promise<Task | undefined> => {
 	const tasks = getTargetTasks(targetTasks);
```

Comment:
If no targets tasks are provided then this function(getTargetTasks) returns all tasks including repository.

So the new logic(&& tasks.includes("repository")) should only evaluate true when a subset of tasks is passed that omits repository